biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -184,7 +184,7 @@ subroutine mortality_rates( cohort_in,bc_in, btran_ft, mean_temp,             &
           if ( (.not. is_decid_dormant) .and. &
                ( btran_ft(cohort_in%pft) <= hf_sm_threshold ) .and. &
                ( ( minval(bc_in%t_soisno_sl) - tfrz ) > soil_tfrz_thresh ) ) then
-             hmort = EDPftvarcon_inst%mort_scalar_hydrfailure(cohort_in%pft)
+             hmort = EDPftvarcon_inst%mort_scalar_hydrfailure(cohort_in%pft)*((hf_sm_threshold- btran_ft(cohort_in%pft))/hf_sm_threshold)
```

Comment:
Should the second part here be capped to be greater than 0 to prevent negative hmort? 

i.e. scaler x ( max(0, ((threshold - btran)/threshold)) )